README.md

```diff
@@ -1,5 +1,5 @@
-# 🤖 Online Chess Cheater Bot
-[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and builds it own chess board. A local stockfish engine then analyzes the position and offers the best move to the user.
+# 🤖 Online Chess Bot
+[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and helps you do a move. A local stockfish engine then analyzes the position and offers the best move to the user.
```

Comment:
how about:
builds a new chess board which indicates the best move to the user.

Reply:
So smth like 
```markdown
# 🤖 Online Chess Cheat Bot
[lichess.org](https://lichess.org/) and [chess.com](https://chess.com) are the two most popular online chess platforms. This application provides the user with the top engine move and position evaluation. Once a live online game is started, it automatically pulls the played moves from the webpage's source and shows on a new local chess board what is the best move to the user.```